Website/app/contact/page.js

```diff
@@ -10,38 +10,44 @@ export default function contact() {
   return (
     <div>
       <Header props={"bg-top"} />
-      <div className="select-none bg-gradient-to-b from-top via-about to-bottom py-5">
-        <h1 className="text-center font-semibold text-7xl">Contact Us</h1>
-        <p className="text-center font-normal pt-5 pb-8 text-lg">
-          We&apos;d love to hear from you!
-        </p>
+      <div className="select-none bg-gradient-to-b from-top via-about to-bottom ">
+      
+        <div className="flex justify-center items-center  w-full max-md:p-2   p-12">
+
         <form
-          className="grid grid-cols-2 justify-items-center"
+          className="md:w-3/5   bg-white rounded-2xl shadow-lg shadow-gray-500 p-14" 
           method="POST"
           action="https://formsubmit.co/e94b201f9a607081e9f9f8ee09ff5e25"
+          aria-label="Contact Us Form"
         >
+                      <h1 className="  max-md:text-3xl mb-4 text-5xl font-bold">Contact Us</h1>
+                      <p className="">          We&apos;d love to hear from you!
+                      </p>
+                      <div className="flex justify-around gap-2 mt-6 max-md:flex-col">
           <input
-            className="bg-white rounded-md focus:outline-blue-600 text-blue-600 outline:none caret-blue-600 md:w-6/12 sm:w-1/2 h-14 p-3 col-span-2"
+            className= "border max-md:w-full rounded-full max-w-[400px] p-2 w-1/2 bg-white  focus:outline-blue-600 text-blue-600 outline:none caret-blue-600 md:w-6/12 sm:w-1/2  h-12  col-span-2"
             type="text"
             name="name"
             autoFocus={true}
             inputMode="text"
             required
             placeholder="Name"
+            pattern="^[A-Za-z\s]{1,}$"
```

Comment:
Allow `'` and `-` for some specific types of names.
```suggestion
            pattern="^[A-Za-z\s'-]{1,}$"
```